### ISSUE TITLE
ALRP

### ISSUE THREAD

**Issue by marshular** [SnailyCAD/snaily-cadv4#408]
### Describe the feature

An in-game ALRP that can pickup plates and get information such as DUIs, BOLOs, etc. Support for these scripts would be cool (https://forum.cfx.re/t/release-wraith-ars-2x-police-radar-and-plate-reader-v1-3-1/1058277 | https://forum.cfx.re/t/prefech-alpr/2653668)

**Comment by Dev-CasperTheGhost** [comment]
How would this connect to SnailyCAD? It sends a request to the CAD from those scripts?

**Comment by marshular** [comment]
Yes it would send a request to see if the plate is wanted or has a BOLO on it like (https://www.youtube.com/watch?v=5oL7Mg6LQgg&ab_channel=SonoranSoftwareSystems)

**Comment by Dev-CasperTheGhost** [comment]
And where would the returned data show up?

**Comment by Dev-CasperTheGhost** [comment]
@marshular progress can be tracked here: https://github.com/SnailyCAD/fivem-integrations, some screenshots

***Note:** this is a work-in-progress*

![image](https://user-images.githubusercontent.com/53900565/154537305-30ec1865-c3ce-4157-8f09-5d6053fff183.png)
![image](https://user-images.githubusercontent.com/53900565/154537324-a0ccc21c-97bd-4b9a-970d-8a8d89c814cd.png)
